<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1292</width>
    <height>653</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RandomLoot's RoomTiler</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitterRoomAndSplitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QScrollArea" name="roomEditorScrollArea">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>69</width>
          <height>612</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitterListAndTileset">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <layout class="QVBoxLayout" name="vlayoutTileButton">
         <item>
          <layout class="QVBoxLayout" name="tilesGridLayout"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="sizeConstraint">
            <enum>QLayout::SetMinimumSize</enum>
           </property>
           <property name="topMargin">
            <number>60</number>
           </property>
           <item>
            <widget class="QPushButton" name="buttonLoadTileset">
             <property name="text">
              <string>Загрузить тайлсет</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <layout class="QHBoxLayout" name="hlayoutTilesetAndParams" stretch="1,0">
         <item>
          <widget class="QScrollArea" name="tilesetEditorScrollArea">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>990</width>
              <height>399</height>
             </rect>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::FieldsStayAtSizeHint</enum>
           </property>
           <property name="horizontalSpacing">
            <number>50</number>
           </property>
           <property name="verticalSpacing">
            <number>30</number>
           </property>
           <property name="leftMargin">
            <number>25</number>
           </property>
           <property name="topMargin">
            <number>25</number>
           </property>
           <item row="1" column="0">
            <widget class="QSpinBox" name="sbTileWidth">
             <property name="maximum">
              <number>100000</number>
             </property>
             <property name="value">
              <number>64</number>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="sbTileHeight">
             <property name="maximum">
              <number>100000</number>
             </property>
             <property name="value">
              <number>64</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QSpinBox" name="sbXoffset"/>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="sbYoffset"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Tile Width</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Tile Height</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Xoffset</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Yoffset</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1292</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="menuRoomNew"/>
    <addaction name="menuRoomLoad"/>
    <addaction name="menuRoomSave"/>
    <addaction name="menuRoomSaveAs"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Операции</string>
    </property>
    <addaction name="menuRoomCodeGet"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="menuRoomNew">
   <property name="text">
    <string>&amp;Новая комната</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="menuRoomLoad">
   <property name="text">
    <string>&amp;Загрузить комнату</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="menuRoomSave">
   <property name="text">
    <string>&amp;Сохранить комнату</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="menuRoomSaveAs">
   <property name="text">
    <string>Сохранить комнату &amp;как</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="menuRoomCodeGet">
   <property name="text">
    <string>&amp;Получить код генерации</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
